wrappers/hnynethttp/nethttp.go

```diff
@@ -45,6 +45,9 @@ func WrapHandler(handler http.Handler) http.Handler {
 			if handlerName != "" {
 				span.AddField("handler.name", handlerName)
 				span.AddField("name", handlerName)
+			} else {
+				// we always want a name, even if it's kinda useless.
+				span.AddField("name", "handler")
```

Comment:
thoughts on just always sending `name: "handler"` (or `"http handler"` or something) and then only sending `handler.name: handlerName` if handlerName has a value?

Reply:
I think there's a lot of value in having a bit more detail in the `name` field; having it always be "handler" feels more like a `type` than a `name`.